### PR TITLE
Added base interceptor

### PR DESCRIPTION
**Pull request by rodrigouz**

### CODE REVIEW

File: version_number [hunk 1]
```diff
@@ -0,0 +1 @@
+2
```
Comment (JimmyRojas) [inline version_number]: Do we need this new file?

File: build.gradle [hunk 2]
```diff
@@ -90,6 +90,12 @@ dependencies {
     //testCompile 'org.spockframework:spock-core:1.0-groovy-2.3'
 }
 
+println "TOKEN:" + "$System.env.EV_TEST"
+
```
Comment (JimmyRojas) [inline build.gradle]: remove new blank lines

**Comment by JimmyRojas** [comment]
Please add UnitTests for the changes

**Comment by rodrigouz** [comment]
Ok, unit test in progress :+1:

**Comment by JimmyRojas** [comment]
Looks good to me..,  thank you
@ziur please take a look

**Comment by ziur** [comment]
:+1:

**Pull request merged**